docs: move about us file and credit contributors (#15661)
renovatebot/renovate: docs/usage/about-us.md

More than 600 outside contributors helped improve Renovate.

## Special thanks

We want to highlight the work of these awesome people.
Thank you for your time and effort!

### Maintainers

Renovate is maintained by:

- [@rarkins](https://github.com/rarkins)
- [@viceice](https://github.com/viceice)
- [@JamieMagee](https://github.com/JamieMagee)

### Maintainers for features

Next up, we have these people who help maintain parts of Renovate:

- [@HonkingGoose](https://github.com/HonkingGoose) master of the docs and community manager
- [@zharinov](https://github.com/zharinov) focused on parsing, Gradle and Maven
- [@secustor](https://github.com/secustor) worked on Terraform and Helm
- [@fgreinacher](https://github.com/fgreinacher) worked on NuGet
- [@Turbo87](https://github.com/Turbo87) has helped in multiple areas, especially Cargo

### Valuable contributions

We want to highlight these valuable contributions, even if they are one-offs.
Some features made a lot of people happy, and efficient!

- [@ikesyo](https://github.com/ikesyo) regularly helpful
- [@astellingwerf](https://github.com/astellingwerf) reviews PRs

> - [@danports](https://github.com/danports) worked on the Flux manager, and other managers. Feel free to ping `@danports` for any Flux-related issue or PR

Author: ```suggestion
- [@danports](https://github.com/danports) worked on the Flux manager, and other managers. Feel free to ping `@danports` for any Flux-related issue or PR
- [@shegox](https://github.com/shegox) worked on the Go manager, and improved our docs
```

> - [@danports](https://github.com/danports) Worked on the Flux manager, and other managers. Feel free to ping `@danports` for any Flux-related issue or PR.

Reviewer: All other sentences have a lower case letter after the username, and don't end with a full stop, so:
```suggestion
- [@danports](https://github.com/danports) worked on the Flux manager, and other managers. Feel free to ping `@danports` for any Flux-related issue or PR
```

## Renovate development

> The source code for Renovate is available on GitHub ([renovatebot/renovate](https://github.com/renovatebot/renovate)).

Author: ```suggestion
The source code for Renovate is available on [GitHub at `renovatebot/renovate`](https://github.com/renovatebot/renovate).
```

This is where we do most of the development.

## About these docs

> The Renovate docs are built from Markdown files in [Renovate's Open Source repos](https://github.com/renovatebot/).

Author: ```suggestion
The Renovate docs are built from Markdown files in our [`renovatebot/renovate` repository](https://github.com/renovatebot/).
```

> Most of the source files can be found in the [main Renovate repository](https://github.com/renovatebot/renovate/tree/main/docs/usage).

Author: ```suggestion
Most of the source files can be found in the [`docs/usage/` directory](https://github.com/renovatebot/renovate/tree/main/docs/usage).
```
